packages/manager/src/features/Linodes/LinodeCreate/Networking/InterfaceGeneration.tsx

```diff
@@ -61,13 +64,51 @@ export const InterfaceGeneration = () => {
           >
             <FormControlLabel
               control={<Radio />}
-              label="Configuration Profile Interfaces (Legacy)"
-              value="legacy_config"
+              label={
+                <Stack mt={1.25} spacing={0.5}>
+                  <Stack direction="row">
+                    <Typography sx={(theme) => ({ font: theme.font.bold })}>
+                      Linode Interfaces
+                    </Typography>
+                    <BetaChip />
+                  </Stack>
+                  <Typography>
+                    Linode Interfaces are the preferred option for VPCs and are
+                    managed directly through a Linode’s Network settings. They
+                    can only be updated, or deleted while the Linode is powered
```

Comment:
```suggestion
                    can only be updated or deleted while the Linode is powered
```
These aren't two separate clauses, if I'm understanding the intent of this message correctly: the Linode must be powered off to update or delete the interface? In that case, we don't need a comma.

Reply:
We remove this part about powering on/off the Linode, so this should no longer be an issue.